libs/liquidity/src/lib/utils/liquidity-utils.ts

```diff
@@ -117,3 +118,46 @@ export const getTargetStake = (
 ) => {
   return markets.find((m) => m.id === marketId)?.data?.targetStake || '0';
 };
+
+export const useCheckLiquidityStatus = ({
```

Comment:
non blocking: similar usage to _TrafficLight_ component from [here](https://github.com/vegaprotocol/frontend-monorepo/blob/15d2aed343b56ec710e5f7d6fce2163f5dea24c1/libs/ui-toolkit/src/components/traffic-light/traffic-light.tsx), may be useful in the future

Reply:
I used Intent because it is used by the Indicator component.